hack/lib/gen_pki.sh

```diff
@@ -16,7 +16,9 @@
 
 set -xue
 
-readonly TEMP_DIR=$(mktemp -d)
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE}")/../.." && pwd -P)"
+KIND_SERVER="$(yq eval '.clusters.[]|select(.name=="kind-kind")|.cluster.server' ~/.kube/config)"
```

Comment:
`kubectl config current-context` to support differently named contexts like below

Reply:
updated!~